sources/Interop/Windows/shared/uuids/Windows.Manual.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from shared/uuids.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace TerraFX.Interop
+{
+    public static partial class Windows
+    {
+        /// <summary>Retrieves the GUID of of a specified type.</summary>
+        /// <typeparam name="T">The type to retrieve the GUID for.</typeparam>
+        /// <returns>A <see cref="UuidOfType"/> value wrapping a pointer to the GUID data for the input type. This value can be either converted to a <see cref="Guid"/> pointer, or implicitly assigned to a <see cref="Guid"/> value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static unsafe UuidOfType __uuidof<T>()
+            where T : unmanaged
+        {
+            return new UuidOfType(UUID<T>.RIID);
+        }
+
+        /// <summary>
+        /// A proxy type that wraps a pointer to GUID data. Values of this type can be implicitly
+        /// converted to and assigned to <see cref="Guid"/>* or <see cref="Guid"/> parameters.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public readonly unsafe ref struct UuidOfType
+        {
+            private readonly Guid* riid;
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We shouldn't need `AggressiveInlining` on these. They are extremely small and the JIT should handle it already.

Reply:
Removed in 65b7f00150d78359e243ead4b46138d2c9624914.